src/Symfony/Component/Notifier/Bridge/Nexmo/NexmoTransport.php

```diff
@@ -73,5 +74,13 @@ protected function doSend(MessageInterface $message): void
                 throw new TransportException(sprintf('Unable to send the SMS: '.$msg['error-text'].' (%s).', $msg['status']), $response);
             }
         }
+
+        $success = $response->toArray(false);
+
+        $message = new SentMessage($message);
+        $message->setMessageId($success['messages'][0]['message-id']);
+        $message->setTransportResponseData($success);
```

Comment:
Passing the returned result as is means that it is not standardized.
I would instead make it a bit more useful by defining a format with things that are common (including the transport name so that one can know how to parse the extra information) and add an entry with the original message.

Reply:
Yes, you are right.

The only real common thing is the message-id. The other data parts depend on :
- the kind of message (chat or SMS)
- the service (some returns cost for SMS, number of SMS sent for the message)

I think we can start with:
- `original message` (in the construct)
- `transport` __toString() (in the construct)
- `message-id` (with setter if returned)

Reply:
Yes, let's not add anything "special" at first. For 5.1, let's be very generic. And I suppose that with me message-id and the transport, you have all the information you need to get more if needed.

Reply:
Yes exactly. Most of the services have webhook or API to get the other information parts.

Thanks.